compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1075,6 +1075,12 @@ TR::Register *OMR::X86::TreeEvaluator::integerStoreEvaluator(TR::Node *node, TR:
    if (instr && node->getOpCode().isIndirect())
       cg->setImplicitExceptionPoint(instr);
 
+   if (node->getSymbol()->isUnsafeShadowSymbol())
```

Comment:
I guess they may use similar, if not the same, strategy by setting GC or exception information at store/load instructions.  However, this evaluator is platform specific.